apps/src/sites/studio/pages/levels/editors/_blockly.js

```diff
@@ -0,0 +1,75 @@
+import $ from 'jquery';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import initializeCodeMirror from '@cdo/apps/code-studio/initializeCodeMirror';
+import initializeBlockPreview from '@cdo/apps/code-studio/initializeBlockPreview';
+import commonBlocks from '@cdo/apps/blocksCommon';
+
+const data = getScriptData('pageOptions');
+// TODO: stop pulling Blockly off of the window object.
+if (window.Blockly && !data.uses_droplet) {
+  window.Blockly.Css.styleSheet_ = window.goog.cssom.addCssText(Blockly.Css.CONTENT.join("\n")).sheet;
+  window.Blockly.assetUrl = path => `/assets/${path}`;
+  const appBlocks = require('@cdo/apps/' + data.app + '/blocks');
+  const options = {
+    skin: {id: data.skin_id},
+    isK1: data.isK1,
+  };
+  commonBlocks.install(window.Blockly, options);
+  appBlocks.install(window.Blockly, options);
+}
+
+const fieldConfig = {
+  startEditor: {
+    codemirrorEl: 'level_start_blocks',
+    blockPreviewEl: 'start-preview',
```

Comment:
nit: Is `El` a standard abbreviation for element? `codemirrorEl` to me looks kind of odd and incomplete. Maybe use `Elem` spell out `Element`, or call it `Id` since these are the element ids. Or even drop the suffix and just call them `codemirror` and `blockPreview`?